umap/views.py

```diff
@@ -937,34 +941,83 @@ class DataLayerUpdate(FormLessEditMixin, GZipMixin, UpdateView):
     model = DataLayer
     form_class = DataLayerForm
 
-    def form_valid(self, form):
-        self.object = form.save()
-        # Simple response with only metadatas (client should not reload all data
-        # on save)
-        response = simple_json_response(
-            **self.object.metadata(self.request.user, self.request)
-        )
-        response["Last-Modified"] = self.last_modified
-        return response
+    def has_been_modified_since(self, if_unmodified_since):
+        if if_unmodified_since and self.last_modified != if_unmodified_since:
```

Comment:
Maybe we could just `return if_unmodified_since and self.last_modified != if_unmodified_since`